core/amber/src/main/scala/edu/uci/ics/texera/web/model/websocket/request/WorkflowExecuteRequest.scala

```diff
@@ -1,13 +1,20 @@
 package edu.uci.ics.texera.web.model.websocket.request
 
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize
 import edu.uci.ics.texera.workflow.common.operators.LogicalOp
 import edu.uci.ics.texera.workflow.common.workflow.{BreakpointInfo, LogicalLink}
 
+case class ReplayExecutionInfo(
+    @JsonDeserialize(contentAs = classOf[java.lang.Long])
+    eid: Long,
```

Comment:
I think eid is `uint`, why do we use `Long` here?

Reply:
on the backend, our `ExecutionIdentity` uses long as `eid` type. Also, `Long` covers `uint`. I think we can use this for now.